src/libs/OptionsListUtils.js

```diff
@@ -284,11 +306,10 @@ function getOptions(reports, personalDetails, draftComments, activeReportID, {
             ))) {
                 return;
             }
-
-            if (searchValue && !isSearchStringMatch(searchValue, personalDetailOption.searchText)) {
+            const {searchText, participantsSet} = personalDetailOption;
```

Comment:
This isn't ideal, but the line will be far too long if I don't do this.

Reply:
What's not idea about it? We can also break if statements onto separate lines if you prefer

Reply:
Ah if you're okay with it I think it's cool, just thought it was weird to destructure an object for the sole purpose of making an if-statement shorter.